README.md

```diff
@@ -193,3 +200,145 @@ For more information on running and tearing down k8s environments, follow this [
 
 Because some of our modules pass headers bigger than 4k we had to increase the default value of the proxy buffer size for nginx.
 We also enable the CORS header for the applications that need it through the Ingress Rule.
+
+# Docker for Desktop - Mac
```

Comment:
We **may** want to target a developer that knows nothing about Docker or Kubernetes here, with nothing installed (except homebrew) as the original script does, with clear, numbered steps with copy and paste commands, starting from downloading Docker for Desktop.

What do others think?

Reply:
I agree with Ray. As a developer not familiar with Docker, the steps to setup the prerequisites in Ray's demo were a great  help.

Reply:
So do we want to hold this ticket over to the next sprint or do we want a new issue to change this in the next sprint?  As is, this meets the acceptance criteria.

Reply:
I agree it meets acceptance criteria, as-is.  If we can't add the additional text quickly and close it, then create a new ticket pls.

Reply:
... and we can close this one